### PR TITLE
Fix video path

### LINKED ISSUE #352: Incorrect video folder
We recently updated the folder structure https://github.com/AllenNeuralDynamics/dynamic-foraging-task/pull/299, but the actual video folder determined in bonsai was not updated.

### PR DESCRIPTION
**Pull request by XX-Yin**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
Updated the video data saving path. 

### What issues or discussions does this update address?
Resolves #352 

### Describe the expected change in behavior from the perspective of the experimenter
The video data will be saved to session/behavior-videos. 

### Describe the outcome of testing this update on a rig in 447
The 447 doesn't have the high-speed camera, tested in 323-EPHYS3.

### CODE REVIEW

File: src/workflows/foraging.bonsai.layout
Comment (hanhou) [inline src/workflows/foraging.bonsai.layout:1]: are changes to this file necessary for this PR?

Comment (XX-Yin) [reply src/workflows/foraging.bonsai.layout:1]: I think part of these changes in the layout file is necessary, but not all of them. Every time I change my workflow, this layout file change automatically likely due to opening different windows. Let me try if I can have some minimal changes.

Comment (hanhou) [reply src/workflows/foraging.bonsai.layout:1]: No worries. As long as you know you didn't accidentally change anything else, you'll be fine.

**Review by hanhou — APPROVED** [review]

File: src/workflows/foraging.bonsai.layout
Comment (XX-Yin) [reply src/workflows/foraging.bonsai.layout:1]: In fact, we can just make some minimal changes to the folder name. I submitted a new PR and will cancel this pull request.

**Pull request closed without merging**